### PR TITLE
[ABSTRACT] abstract supplied

### PR DESCRIPTION
**Pull request by MattSScott**

### CODE REVIEW

File: 000_introduction/000_introduction.tex
Comment (mdubied) [inline 000_introduction/000_introduction.tex]: This work can probably not be referred as a paper (I'm not sure which journal would accept a 150 pages long paper hahah). Maybe simply use "project" or "work".

Comment (mdubied) [inline 000_introduction/000_introduction.tex]: In the last sentence, maybe "...Through the integration of governance _using treaties_"?

**Review by mdubied — COMMENTED** [review]
Knowing the content of the abstract, I would maybe give a few words about the platform problem on a non-theoretical level in the introduction. But you had a meeting about this I think, so maybe you can quickly explain me where an overview of the simulation setup will be given to the reader

File: 000_introduction/000_introduction.tex
Comment (MattSScott) [reply 000_introduction/000_introduction.tex]: ahh nice spot - force of habit haha

Comment (MattSScott) [reply 000_introduction/000_introduction.tex]: sounds good - like it :)

**Review by fds101 — DISMISSED** [review]
Feel like @mdubied covered everything I saw. Approved from me pending the changes proposed.

**Comment by MattSScott** [comment]
Depends the most on @udai-arneja and @priyachh - will await either of their approvals before merging

**Comment by MattSScott** [comment]
> Knowing the content of the abstract, I would maybe give a few words about the platform problem on a non-theoretical level in the introduction. But you had a meeting about this I think, so maybe you can quickly explain me where an overview of the simulation setup will be given to the reader

Great question: there is a whole section dedicated to both the scenario and the simulator. We chose to keep the intro and abstract purely theoretical and offer background instead

**Comment by mdubied** [comment]
> > Knowing the content of the abstract, I would maybe give a few words about the platform problem on a non-theoretical level in the introduction. But you had a meeting about this I think, so maybe you can quickly explain me where an overview of the simulation setup will be given to the reader
> 
> Great question: there is a whole section dedicated to both the scenario and the simulator. We chose to keep the intro and abstract purely theoretical and offer background instead

It is a choice I can follow. As long as it was discussed with those who write the corresponding section, this is fine for me

File: main.tex
Comment (jzzheng22) [inline main.tex]:
```suggestion
Co-operative survival games refer to a subset of political choice games wherein `players' must work together to overcome disaster, or else suffer the consequences through both personal and communal damage. Furthermore, without the ability to monitor a system and in the absence of a centralised authority, it becomes impossible to form enduring self-governing institutions, resulting in the proposed self-organising mechanisms for `solving' such games potentially functioning in harmful ways. This project utilises a self-organising multi-agent system to simulate an iterated collective action, co-operative survival game comprising a resource management problem, the `solution' to which involves a system of localised governance. Through a series of experiments where the behaviour of the players in the system is increasingly randomised and the presence of self-organising mechanisms is toggled, we investigate the causes of instability. Ultimately we conclude that through the integration of governance using treaties, it is possible for such a system to effectively self-organise to minimise the total deaths of the players involved.
```

**Review by jzzheng22 — CHANGES_REQUESTED** [review]
Thanks Matt :) I like it

File: main.tex
Comment (MattSScott) [reply main.tex]: using a comma before "disaster, else" negates the need for an "or"

also drop-in-clause would be needed if we're keeping "using treaties, it is" or else we remove both

**Comment by priyachh** [comment]
I quite like this.  @udai-arneja what do you think?

File: main.tex
Comment (MattSScott) [reply main.tex]: Overall definitely agree with your second change though, made a slight change :)

**Review by jzzheng22 — APPROVED** [review]

**Comment by udai-arneja** [comment]
@MattScottEEE didnt mention the honesty aspect we discussed, but tbh after reading it I think this isn't the place for it - I'll discuss in the assumptions section. Really like this abstract, thanks for this.

**Review by udai-arneja — APPROVED** [review]

**Review by priyachh — APPROVED** [review]

**Comment by MattSScott** [comment]
Thanks everyone - really helpful stuff from all <3

**Pull request merged**
